### ISSUE TITLE
voice chat audio playback not loading on safari and other browsers on ios device

### ISSUE THREAD

**Issue by lydenn** [chatwoot/chatwoot#5208]
**Describe the bug**

unable to play the audio, the audio playback is now loading.

**Screenshots**

reply box
<img width="965" alt="Screen Shot 2022-08-05 at 11 43 36 AM" src="https://user-images.githubusercontent.com/102777811/183003196-29d5d196-e04d-4e51-93b8-5de9bf4a4348.png">

message audio playback
<img width="310" alt="Screen Shot 2022-08-05 at 11 43 47 AM" src="https://user-images.githubusercontent.com/102777811/183003258-2e547d80-1b95-41ab-bc81-16407920683e.png">

**Environment**
self hosted chatwoot/Linux VM installation

**Desktop (please complete the following information):**
 - OS: masOS
 - Browser Safari

**Smartphone (please complete the following information):**
 - Device: Apple device
 - Browser safari, chrome

**Comment by giquieu** [comment]
@lydenn Can you report the browser version?

**Comment by lydenn** [comment]
@giquieu yes, the comment is updated

**Comment by lydenn** [comment]
i think these can be the solutions

1. use vmsg instead of opus recorder: https://collab-project.github.io/videojs-record/#/plugins/vmsg
2. or we can still use the opus recorder and convert the output to mp3 format by using ffmpeg: https://collab-project.github.io/videojs-record/#/plugins/ffmpeg.wasm

**Comment by giquieu** [comment]
Hello, 
Could you add the browser console log?
Another interesting validation, check if there is availability of Media in the Browser, run this and see the result:
navigator.mediaDevices.getUserMedia({audio: true})

Some browsers have discontinued the ScriptProcessorNode which is being used in Wavesurfer, maybe that's your problem. I'm following some issues in the repositories of the libs used...

I've already done some tests with the Libs you showed, maybe I can reevaluate and bring new results.

**Comment by lydenn** [comment]
@giquieu i run this navigator.mediaDevices.getUserMedia({audio: true}) and the microphone work just fine.

i think it's the problem of audio file format that we currently use (audio/ogg), safari and ios devices no longer support this file format. 
if you have already tried the vmsg plugin, can you share me some example of that since i cannot make the video record work with the vmsg plugin

**Comment by muhsin-k** [comment]
@giquieu @lydenn 
> @giquieu i run this **navigator.mediaDevices.getUserMedia({audio: true})** and the microphone work just fine.
> 
> i think it's the problem of audio file format that we currently use (audio/ogg), safari and ios devices no longer support this file format. 
Yes. The Safari browser doesn't support `ogg` format audio files. 
> if you have already tried the vmsg plugin, can you share me some example of that since i cannot make the audio record work with the vmsg plugin
Did you able to fix this problem?

**Comment by giquieu** [comment]
Hello @muhsin-k 
I was validating this. The divergence is occurring in the wavesurfer.
If there's no problem, I can continue...

**Comment by muhsin-k** [comment]
@giqu Yes. You can continue on this. It seems like we need to convert it to an `mp3` file to support all browsers.
